.github/workflows/docker-publish.yml

```diff
@@ -67,6 +67,7 @@ jobs:
         images: ${{ inputs.registry }}/${{ github.repository_owner }}/${{ matrix.target }}
         tags: |
           type=ref,event=pr
+          type=ref,event=branch,enable=${{ github.ref_name != 'main' }}
```

Comment:
I don't understand this conditional. What is this line doing? I am probably misunderstanding it because we _do_ want to enable the publish on `main` right?

Reply:
Yes, but for main we call the tag `edge` by convention, which is defined on the line just below. We could omit this conditional and then have both `edge` and `main` tags, or remove the `edge` tag. I don't really care much, but it would be nice to keep it consistent with aiidalab-docker-stack.